theolive/playback/web/basic-playback-guide.mdx

```diff
@@ -1,7 +1,11 @@
+---
+title: Basic Playback Guide
```

Comment:
Nit-nit-pick: you don't need to specify a `title` in the frontmatter if you already put it in a level 1 heading in the Markdown. 😉

Reply:
Agreed, we can review all the pages once we have the structure in place and clean.